test/integration/push_int_test.go

```diff
@@ -161,12 +150,12 @@ func (suite *PushIntegrationTestSuite) TestCarlisle() {
 
 	prj, err := project.FromPath(filepath.Join(wd, constants.ConfigFileName))
 	suite.Require().NoError(err, "Could not parse project file")
-	suite.Assert().True(prj.IsHeadless(), "project should be headless: URL is %s", prj.URL())
+	suite.Assert().False(prj.IsHeadless(), "project should NOT be headless: URL is %s", prj.URL())
 
 	ts.LoginAsPersistentUser()
 
 	// convert to real project
-	cp = ts.SpawnWithOpts(e2e.WithArgs("init", namespace), e2e.WithWorkDirectory(wd))
+	cp = ts.SpawnWithOpts(e2e.WithArgs("init", namespace, "python3"), e2e.WithWorkDirectory(wd))
```

Comment:
I think this _may_ cause a test failure. As the base project that is activated in this test can either by Perl-5.32 or Python. It is only a Python project on MacOS